extensions/amp-sidebar/0.1/amp-sidebar.js

```diff
@@ -160,7 +177,9 @@ export class AmpSidebar extends AMP.BaseElement {
         }, ANIMATION_TIMEOUT);
       });
     });
-    this.getHistory_().push(this.close_.bind(this)).then(historyId => {
+    this.getHistory_().push(() => {
+      this.close_();
```

Comment:
I didn't understand the issue with event sequence. What's the exact event sequence in this case?